README.md

```diff
@@ -1,147 +1,68 @@
 
 # react-native-zoom-us
 
-This is a minimum bridge of https://github.com/zoom/zoom-sdk-android and https://github.com/zoom/zoom-sdk-ios
-
-Tested on XCode 9.4.1 and node 10.14.1.
+This is a bridge for ZoomUS SDK:
+- android: https://github.com/zoom/zoom-sdk-android
+- ios: https://github.com/zoom/zoom-sdk-ios
 
 Pull requests are welcome.
 
+- [Example](https://github.com/mieszko4/react-native-zoom-us-test)
```

Comment:
I need to update this example - it is very old

Reply:
Ok. I just moved link from the bottom.